rulesets/README.md

```diff
@@ -0,0 +1,28 @@
+# Rulesets for opensafely-actions repositories
+
+
+This directory contains several [rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets)
+for repositories in the opensafely-actions organisation.
+
+## Importing a ruleset into a repository
+Follow [the instructions in the GitHub docs for importing a ruleset](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/managing-rulesets-for-a-repository#importing-a-ruleset).
+
+Note that the [Protect default branch ruleset](#protect-default-branch-protect-default-branchjson) requires status
```

Comment:
The link is broken. But the document is so short that the link isn't necessary. Let's remove it.

```suggestion
Note that the Protect default branch ruleset requires status
```